docs/addresses.rst

```diff
@@ -5,9 +5,10 @@ In IOTA, addresses are generated deterministically from seeds. This
 ensures that your account can be accessed from any location, as long as
 you have the seed.
 
-Note that this also means that anyone with access to your seed can spend
-your IOTAs! Treat your seed(s) the same as you would the password for
-any other financial service.
+.. warning::
+  Note that this also means that anyone with access to your seed can spend
+  your IOTAs! Treat your seed(s) the same as you would the password for
```

Comment:
❔ `IOTAs` or `iotas`?  Did we ever get to the bottom of that? 😁

Reply:
not really :D I will write `iotas` from now on.